lib/Dialog/Dialog.js

```diff
@@ -144,7 +154,9 @@ module.exports = kind(
 			]}
 		]},
 		{kind: Divider, classes: 'moon-dialog-divider'},
-		{name: 'message', kind: BodyText, classes: 'moon-dialog-content'},
+		{name: 'bodyComps', components: [
```

Comment:
hi @KunmyonChoi . Yes but the problem would be `BodyText ` kind. It doesn't add any components but it will add classes and extra properties which will be applied on DOM. So I adopted this approach.